UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -370,7 +376,13 @@ private void UpdateMe()
 		{
 			if (NetworkTime.time >= CountdownEndTime)
 			{
-				StartRound();
+				if (PlayerList.Instance.ReadyPlayers.Count >= MinReadyPlayersForCountdown)
+				{
+					StartRound();
+				}
+				else {
+					Logger.Log("More ready players needed for countdown!")
+				}
```

Comment:
Found the tests failing issue : )
Missing semicolon.

```suggestion
```

I've opted to just remove the log instead of fixing it as it is somewhat expected behaviour and could add quite a few entries to the log.